telegram/dispatcher.py

```diff
@@ -635,4 +647,7 @@ def call_handler(self, handler, update, **kwargs):
         if is_async or 'groupdict' in fargs:
             target_kwargs['groupdict'] = kwargs.get('groupdict', None)
 
+        if 'context' in fargs:
```

Comment:
Please use `if is_async or 'context' in fargs:` here, since async handlers must always receive all kwargs